data/en/getapplicationmetadata.json

```diff
@@ -12,8 +12,14 @@
 		"lucee": {"minimum_version":"", "notes":"", "docs":"https://docs.lucee.org/reference/functions/getapplicationmetadata.html"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/getapplicationmetadata"}
 	},
-	"links": [
-
-	]
+	"examples" : [
+		{
+			"title": "Simple Example",
+			"description": "",
+			"code": "// Fetch application meta data\r\ndata = getApplicationMetadata();\r\n\r\n// Print application name\r\nwriteOutput(\"Application name is \" & (data.name.length() ? data.name : \"unspecified\") & \"<br>\");\r\n\r\n// Print session timeout\r\nwriteOutput(\"Session timeout is \" & data.sessionTimeout & \"<br>\");\r\n\r\n// Print session management\r\nwriteOutput(\"Session management is \" & (data.sessionManagement ? \"enabled\" : \"disabled\"));",
+			"result": "Prints the above statements using application meta data."
```

Comment:
The `result` string should be used for the actual computed result, so in this case I would just leave the result empty, and move `Prints the above statements using application meta data.` to the `description`.